.github/actions/check_for_stc/action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Ensure STC is installed with Pyccel'
+
+runs:
+  using: "composite"
+  steps:
+    - name: check stc installation
+      shell: python
+      run: |
+        import pyccel
+        from os.path import exists, join
+        pyccel_loc = pyccel.__path__[0]
+        print(pyccel_loc)
+        assert(exists(join(pyccel_loc,'extensions/STC/include/stc')))
```

Comment:
Please can you add a few lines to this action to make sure that we don't install the docs. I don't want them creeping back in in the future.

Reply:
Fixed : dd1abd8